### ISSUE TITLE
Blank ids when parsing style in SSR

### ISSUE THREAD

**Issue by thangngoc89** [tkh44/emotion#88]
- `emotion` version: 5.1.0
- `react` version: 15.6.1

Relevant code.

```javascript
import { extractCritical } from "emotion/server"
const { html, ids, css } = extractCritical(<App />)

assert.deepEqual(ids, []) // true
```

What you did:
SSR rendering follows docs

What happened:
blank ids

Reproduction:

Problem description:

Suggested solution:

As I can see in snapshot test of server.js, ids is blank too https://github.com/tkh44/emotion/blob/633cf73e4fbdf266ae5e614289faf5abe16a66bf/test/__snapshots__/server.test.js.snap#L26

**Comment by mitchellhamilton** [comment]
Closed by #89.

**Comment by tkh44** [comment]
@thangngoc89 released in 5.1.1

Thanks @mitchellhamilton
